install/install.ps1

```diff
@@ -54,7 +54,10 @@ if (Test-Path -Path $latestDirectory) {
 
 # We use a directory junction here because not all Windows users will have permissions to create a symlink.
 Write-Verbose "Linking ${latestDirectory} to ${extractedDirectory}"
-$null = New-Item -Path $latestDirectory -Target $extractedDirectory -ItemType Junction
+cmd.exe /Q /C "mklink /J $latestDirectory $extractedDirectory" > $null
+if (!$?) {
+    Write-Error "Linking failed!"
+}

```

Comment:
Would you mind explaining why the old version 
```ps1
$null = New-Item -Path $latestDirectory -Target $extractedDirectory -ItemType Junction
```
does not work?

For the newer version, what is the difference between
```ps1
cmd.exe /Q /C "mklink /J $latestDirectory $extractedDirectory" > $null
```
and
```ps1
mklink /J $latestDirectory $extractedDirectory > null
```

Reply:
I've updated the PR Description to address this